<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preDialog</class>
 <widget class="QDialog" name="preDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>634</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Slice Preview</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/imgs/sew1.png</normalon>
   </iconset>
  </property>
  <widget class="QLabel" name="pWindow1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>45</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_close">
   <property name="geometry">
    <rect>
     <x>1015</x>
     <y>600</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Hack</family>
     <pointsize>8</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="pWindow2">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>45</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="pSliderView">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>565</y>
     <width>1005</width>
     <height>30</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QSlider" name="pSlider_i1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>44</y>
     <width>16</width>
     <height>480</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:vertical{
    background: rgb(255, 255, 255);	
    border: 2px solid rgb(128, 128, 128);
    width: 10px;
    border-radius: 3px;
	 margin-left: 7px;
	margin-right: 7px;
}
QSlider::add-page:vertical {
background: rgb(0, 0, 0);
border: 2px solid #fff;

border-radius: 5px;
margin-left: 7px;
margin-right: 7px;

}

QSlider::sub-page:vertical {
background: rgb(255, 0, 0);
border: 2px solid #fff;

border-radius: 5px;
margin-left: 7px;
margin-right: 7px;

}

QSlider::handle:vertical {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 rgb(150, 150, 150), stop:1 rgb(100, 100, 100));
border: 1px solid #777;
height: 12px;
margin-top: -3px;
margin-bottom: -3px;
margin-left: -3px;
margin-right: -3px;

border-radius: 3px;

}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSlider" name="pSlider_i2">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>44</y>
     <width>16</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:vertical{
    background: rgb(255, 255, 255);	
    border: 2px solid rgb(128, 128, 128);
    width: 10px;
    border-radius: 3px;
	 margin-left: 7px;
	margin-right: 7px;
}
QSlider::add-page:vertical {
background: rgb(255, 0, 0);
border: 2px solid #fff;

border-radius: 5px;
margin-left: 7px;
margin-right: 7px;

}

QSlider::sub-page:vertical {
background: rgb(0, 0, 0);
border: 2px solid #fff;

border-radius: 5px;
margin-left: 7px;
margin-right: 7px;

}

QSlider::handle:vertical {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 rgb(150, 150, 150), stop:1 rgb(100, 100, 100));
border: 1px solid #777;
height: 12px;
margin-top: -3px;
margin-bottom: -3px;
margin-left: -3px;
margin-right: -3px;

border-radius: 3px;

}</string>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="invertedAppearance">
    <bool>true</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="pSB_i1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>525</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="pSB_i2">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>525</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>600</y>
     <width>916</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="pL_min">
      <property name="font">
       <font>
        <family>Hack</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>min:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="pSB_min">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="pL_bin">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Hack</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>binarization threshhold:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="pSB_bin">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>170</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>170</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="pL_max">
      <property name="font">
       <font>
        <family>Hack</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>max:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="pSB_max">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="pSlider_FMscale">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>15</y>
     <width>806</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal{
    background: rgb(255, 255, 255);	
    border: 2px solid rgb(128, 128, 128);
    height: 10px;
    border-radius: 3px;
 margin-top: 7px;
margin-bottom: 7px;
}
QSlider::add-page:horizontal {
background: rgb(0, 0, 0);
border: 2px solid #fff;

border-radius: 5px;
margin-top: 7px;
margin-bottom: 7px;

}

QSlider::sub-page:horizontal {
background: rgb(255, 0, 0);
border: 2px solid #fff;

border-radius: 5px;
margin-top: 7px;
margin-bottom: 7px;

}

QSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 rgb(150, 150, 150), stop:1 rgb(100, 100, 100));
border: 1px solid #777;
width: 12px;
margin-top: -3px;
margin-bottom: -3px;
margin-left: -3px;
margin-right: -3px;

border-radius: 3px;

}</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>66</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="pSB_FMscale">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>10</y>
     <width>46</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="pL_FMscale">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>15</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FM ROI size [%]:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pButton_close</sender>
   <signal>clicked()</signal>
   <receiver>preDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>997</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>887</x>
     <y>534</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pSB_i1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pSlider_i1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pSB_i2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pSlider_i2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1092</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>1097</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pSlider_i1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pSB_i1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>500</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pSlider_i2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pSB_i2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1097</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>1092</x>
     <y>500</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pSlider_FMscale</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pSB_FMscale</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>612</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>1037</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pSB_FMscale</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pSlider_FMscale</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1037</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>612</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
